packages/@webex/plugin-presence/src/presence.ts

```diff
@@ -0,0 +1,266 @@
+    if (payload.type && payload.payload) {
+      this.trigger(event, payload);
+    }
+  },
+
+  /**
+   * Enables presence feature
+   * @returns {Promise<boolean>} resolves with true, if successful
+   */
+  enable(): Promise<boolean> {
+    return this.webex.internal.feature
+      .setFeature(USER, USER_PRESENCE_ENABLED, true)
+      .then((response) => response.value);
+  },
+
+  /**
+   * Disables presence feature
+   * @returns {Promise<boolean>} resolves with false, if successful
+   */
+  disable(): Promise<boolean> {
+    return this.webex.internal.feature
+      .setFeature(USER, USER_PRESENCE_ENABLED, false)
+      .then((response) => response.value);
+  },
+
+  /**
+   * Returns true if presence is enabled, false otherwise
+   * @returns {Promise<boolean>} resolves with true if presence is enabled
+   */
+  isEnabled(): Promise<boolean> {
+    return this.webex.internal.feature.getFeature(USER, USER_PRESENCE_ENABLED);
+  },
+
+  /**
+   * The status object
+   * @typedef {Object} PresenceStatusObject
+   * @property {string} url: Public resource identifier for presence
+   * @property {string} subject: User ID for the user the returned composed presence represents
+   * @property {string} status: Current composed presence state
+   * @property {string} statusTime: DateTime in RFC3339 format that the current status began
+   * @property {string} lastActive: DateTime in RFC3339 format that the service last saw activity from the user.
+   * @property {string} expires: DEPRECATED - DateTime in RFC3339 format that represents when the current
+   * status will expire. Will not exist if expiresTTL is -1.
+   * @property {Number} expiresTTL: TTL in seconds until the status will expire. If TTL is -1 the current
+   * status has no known expiration.
+   * @property {string} expiresTime: DateTime in RFC3339 format that the current status will expire. Missing
+   * field means no known expiration.
+   * @property {Object} vectorCounters: Used for packet ordering and tracking.
+   * @property {Boolean} suppressNotifications: Indicates if notification suppression is recommended for this status.
+   * @property {string} lastSeenDeviceUrl: Resource Identifier of the last device to post presence activity for
+   * this user.
+   */
+
+  /**
+   * Gets the current presence status of a given person id
+   * @param {string} personId
+   * @returns {Promise<PresenceStatusObject>} resolves with status object of person
+   */
+  get(personId: string): Promise<IPresenceStatusObject> {
+    if (!personId) {
+      return Promise.reject(new Error('A person id is required'));
+    }
+
+    return this.webex
+      .request({
+        method: 'GET',
+        service: 'apheleia',
+        resource: `compositions?userId=${personId}`,
+      })
+      .then((response) => response.body);
+  },
+
+  /**
+   * @typedef {Object} PresenceStatusesObject
+   * @property {Array.<PresenceStatusObject>} statusList
+   */
```

Comment:
Should we consider converting this to a type?

Reply:
It's already converted and added.

Reply:
So, we are holding this just for documentation purposes? If so, I think we have a separate ticket to get a separate API reference like the Calling and EAF SDK for this.

Reply:
I kept them so that we don't lose anything. Maybe we can remove them when working on the API reference ticket.